tol_lab_share/message_properties/definitions/final_nano_drop.py

```diff
@@ -0,0 +1,15 @@
+from .message_property import MessageProperty
+from typing import List, Callable
+
+
+class FinalNanoDrop(MessageProperty):
+    """MessageProperty subclass to manage parsing of a valid Final NanoDrop string provided by another
+    MessageProperty.
+    The Final NanoDrop has to be a string.
+    Eg: '100'
+    """
+
+    @property
+    def validators(self) -> List[Callable]:
+        """Defines the list of validators"""
+        return [self.check_is_string]
```

Comment:
Are these string validators needed?  The schema prevents it being anything other than a string.